src/_pytest/config/__init__.py

```diff
@@ -1072,6 +1073,14 @@ def _checkversion(self):
                     )
                 )
 
+    def _validatekeys(self):
+        for key in self._get_unknown_ini_keys():
+            sys.stderr.write("WARNING: unknown config ini key: {}\n".format(key))
```

Comment:
i wonder if we should use  real python warning here and set the file to the configfile/the line to the line of the key

Reply:
I used `sys.stderr` becauase it looks like the rest of this file does ( see https://github.com/pytest-dev/pytest/blob/master/src/_pytest/config/__init__.py#L1228 for example ). I don't mind changing it though